zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinClientIntegrationTest.java

```diff
@@ -80,7 +80,7 @@ static void destroy() throws Exception {
 
   @BeforeEach
   void setup() {
-    conf = zepServer.getZeppelinConfiguration();
+      zConf = zepServer.getZeppelinConfiguration();
```

Comment:
Please correct the indentation.

```suggestion
    zConf = zepServer.getZeppelinConfiguration();
```